src/style.css

```diff
@@ -0,0 +1,150 @@
+body {
+  background-color: #efefef;
+}
+
+.list {
+  box-shadow: 2px 2px 2px 2px rgb(0 0 0 / 10%);
+  background-color: #fff;
+  margin: auto;
+  margin-top: 100px;
+  width: 60%;
+}
+
+.add-to-list {
+  display: flex;
+  flex-direction: row;
+  justify-content: space-between;
+}
+
+.to-do {
+  font-family: 'Lucida Sans', 'Lucida Sans Regular', 'Lucida Grande', 'Lucida Sans Unicode', Geneva, Verdana, sans-serif;
```

Comment:
Since your page uses only one font type, consider setting the font property on the body of the document. This applies it to all elements on the page. 
```
body {
  font-family: 'Lucida Sans', 'Lucida Sans Regular', 'Lucida Grande', 'Lucida Sans Unicode', Geneva, Verdana, sans-serif;
}
```
You could also create a font variable or class and apply that to the relevant elements. 
Either of the above prevents you from repeating the same line of code.